migrations/20160103164746_set_date_time_fields_with_previous_default_to_null.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class SetDateTimeFieldsWithPreviousDefaultToNull extends AbstractMigration
+{
+    /**
+     * @var array
+     */
+    private $tables = [
+        'talks' => [
+            'created_at',
+            'updated_at',
+        ],
+        'users' => [
+            'activated_at',
+            'created_at',
+            'updated_at',
+            'last_login',
+        ],
+    ];
+
+    public function up()
+    {
+        $this->run('UPDATE `%s` SET `%s` = NULL WHERE `%s` = "1000-01-01 00:00:00"');
```

Comment:
Found some stuff... but... I kinda want to pull this down and try it out as I have a good way of reproducing on previous versions of MySQL now; instead of thrashing a bit. Technically, you should be able to have that query run by ignoring the warning generated by `NO_ZERO_DATE` SQL mode i.e.:

```sql
UPDATE IGNORE `%s` SET `%s` = NULL WHERE `%s` IN ("1000-01-01 00:00:00"', "0000-00-00 00:00:00"')
```

But it's a bit of a hack. I think it's okay in this one-time case because we simply need to query by that value to be able to `NULL` its value.